holidayextras/ui-toolkit
# HELLO - Is it me you're looking for?

#### What does this PR do? (please provide any background)
- Removes a random hardcoded HELLO
- Removes the duplication of a `label` element. By removing the one from `<input>` and surrounding and putting it before the stepper itself. This is better for usability purposes and styling.
- This does introduce a very strange A11Y warning

#### What tests does this PR have?
Checks a label is present when and ID is passed through

#### How can this be tested?
run `npm run docs` click view the stepper component and click on the label (now `number of passengers` rather than `HELLO`) it should enter the input element (which is now NOT read-only by default)

#### Screenshots / Screencast
<img width="854" alt="screen shot 2015-11-25 at 17 00 41" src="https://cloud.githubusercontent.com/assets/778942/11403681/146eb7aa-9396-11e5-921e-f11a9c4d2684.png">

#### What gif best describes how you feel about this work?
![HELLO](https://media.giphy.com/media/14haWnUbgCax2M/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
